.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
How can we solve this to not have on master? I put this in my .gitignore just as fail proof.